wgpu/utils/__init__.py

```diff
@@ -15,3 +15,4 @@
 
 from ._device import get_default_device  # noqa: F401
 from ._compute import compute_with_buffers  # noqa: F401
+from ._shadertoy import Shadertoy  # noqa: F401
```

Comment:
`_shadertoy.py` imports `wgpu.backends.rs`, which is something the user should do (see the "Usage" section of the README).

I would recommend to just remove this import here in to avoid the problem :) it's an optional util, people can import the ShaderToy class if they want to use it

Reply:
Has been modified. 😄